packages/fxa-settings/src/pages/Signup/PrimaryEmailVerified/mocks.tsx

```diff
@@ -0,0 +1,5 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export const MOCK_SERVICE = 'Example Service';
```

Comment:
I think we'll need to update this after I merge in the `signin` PR, because that introduces an enum for the service names, but this looks closer to merging than that is, so I'll introduce the change there!